server/src/main/java/org/opensearch/index/SegmentReplicationPressureService.java

```diff
@@ -65,13 +66,20 @@ public class SegmentReplicationPressureService implements Closeable {
         Setting.Property.NodeScope
     );
 
-    public static final Setting<TimeValue> MAX_REPLICATION_TIME_SETTING = Setting.positiveTimeSetting(
+    public static final Setting<TimeValue> MAX_REPLICATION_TIME_BACKPRESSURE_SETTING = Setting.positiveTimeSetting(
```

Comment:
can you pls add a comment here on the difference between this and `MAX_REPLICATION_LIMIT_STALE_REPLICA_SETTING`

Reply:
Sure.